pages/modal/scroll-padding.page.tsx

```diff
@@ -0,0 +1,40 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import React, { useState } from 'react';
+import ScreenshotArea from '../utils/screenshot-area';
+import { Button, Modal } from '~components';
+
+export default function () {
+  const [visible, setVisible] = useState(false);
+
+  return (
+    <article>
+      <h1>Scroll padding modal</h1>
+      <Button data-testid="modal-trigger" onClick={() => setVisible(true)}>
+        Show modal
+      </Button>
+      <ScreenshotArea>
```

Comment:
why are doing screenshot test on this page?

Reply:
Not really necessary, just copied over from other pages. I can remove this.

Reply:
Done